lib/tasks/support.rake

```diff
@@ -77,7 +77,7 @@ namespace :support do
       if active_subscriptions.empty?
         puts "Subscriber #{email_address} already unsubscribed from #{subscriber_list_slug}"
       else
-        UnsubscribeService.call(subscriber, [active_subscriptions.last], :unsubscribed)
+        active_subscriptions.last.end(:unsubscribed)
```

Comment:
Not your issue but it's a bit strange we do a `.last` here. It feels like https://github.com/alphagov/email-alert-api/blob/614fac637709cda9100fdca96e37e374d67081c9/lib/tasks/support.rake#L76

should be `subscription = Subscription.active.find_by(subscriber_list: subscriber_list, subscriber: subscriber)`

Reply:
Good point, I've adjusted it to use `find_by`.